moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast.h

```diff
@@ -306,8 +305,6 @@ class IkSolutionList : public IkSolutionListBase<T>
 };
 }  // namespace ikfast
 
-#endif  // OPENRAVE_IKFAST_HEADER
-
```

Comment:
I think you removed the wrong lines here and that's the problem with this PR.  The `#endif` you want to remove is at the end of this file.

Reply:
I fixed this.